lib/install.js

```diff
@@ -55,58 +58,59 @@ async function installForPlatform (version, platform, arch) {
 
   const url = getDownloadUrl(version, platform, arch);
 
-  log.info(`Installing Chromedriver version '${version}' for platform '${platform}' and architecture '${arch}'`);
+  log(`Installing Chromedriver version '${version}' for platform '${platform}' and architecture '${arch}'`);
 
   // set up a temp file to download the chromedriver zipfile to
-  let binarySpec = `chromedriver_${platform}${arch}`;
-  log.info(`Opening temp file to write ${binarySpec} to...`);
-  let tempFile = await tempDir.open({
+  const binarySpec = `chromedriver_${platform}${arch}`;
+  log(`Opening temp file to write '${binarySpec}' to...`);
+  const tempFile = await tempDir.open({
```

Comment:
I'd rather create a temporary folder (`openDir`) and put all the stuff there. Then delete it in `finally` section to make sure we don't clutter user's temporary file system much